sequencing/vcf_qc/dxapp.json

```diff
@@ -102,62 +175,39 @@
       "label": "Additional Hard Filters",
       "class": "string",
       "optional": true
-    },
-    {
-      "group": "Build Options",
-      "name": "build_version",
-      "label": "NCBI Build Version",
-      "class": "string",
-      "choices": ["rgc_b38","gatk_b38","b37_decoy","hg38chr.fa"],
-      "default": "rgc_b38",
-      "optional": false
     }
   ],
   "outputSpec": [
     {
-      "name": "vcf_out",
-      "label": "Filtered VCF file",
+      "name": "out_vcfgz",
+      "label": "Filtered VCF files",
       "class": "array:file",
-      "patterns": [
-        "*.vcf.gz"
-      ]
+      "patterns": ["*.vcf.gz"]
     },
     {
-      "name": "vcfidx_out",
-      "label": "Filtered VCF Index file",
+      "name": "out_vcfidx",
+      "label": "Filtered VCF Index files",
       "class": "array:file",
-      "patterns": [
-        "*.tbi"
-      ]
+      "patterns": ["*.tbi", "*.vcf.gz.tbi"]
     }
   ],
   "runSpec": {
-  "execDepends": [
-      {
-        "name": "pigz"
-      },
-      {
-        "name": "tabix"
-      }
-    ],
-    "systemRequirements": {
-      "*": {
-        "instanceType": "mem1_ssd2_x8"
-      }
-    },
     "distribution": "Ubuntu",
     "release": "14.04",
+    "restartableEntryPoints": "all", 
     "interpreter": "bash",
-    "file": "src/vcf_qc.sh"
+    "file": "src/vcf_qc.sh",
+    "timeoutPolicy": {"*": { "hours": 72 }}
   },
-  "resources" : "project-BYpFk1Q0pB0xzQY8ZxgJFv1V",
-  "access": {
-    "network": [
-      "*"
-    ],
-     "allProjects": "VIEW"
+  "regionalOptions": {
+    "aws:us-east-1": {"systemRequirements": {"*": {"instanceType": "mem1_ssd2_x8"}},
+                    "assetDepends": [{"id": "record-FGpf7zQ0kfbZxj6q4ZzQ2BPG"}]},
+    "azure:westus": {"systemRequirements": {"*": {"instanceType": "azure:mem2_ssd1_x16"}},
+                    "assetDepends": [{"id": "record-FGpfZ1j95qzXV43bP5y5JKgV"}]}
```

Comment:
Please update record IDs to:
aws:us-east-1: record-FJ3zf8j08zVZFzqYG88Pbk7z
azure:westus: record-FJ4058j9Vf9ZBBVB2Q7b3VQz